src/pages/who-we-are/components/_hero.tsx

```diff
@@ -118,19 +121,19 @@ const Hero = () => {
                             alt="example"
                             width="unset"
                         />
-                        <StyledH1>{title}</StyledH1>
+                        <Flex jc="center" p="0 32px" max_width="1440px">
+                            <DesktopHeader>{title}</DesktopHeader>
+                        </Flex>
                     </StyledFlex>
                 </DesktopWrapper>
                 <MobileWrapper>
```

Comment:
Can the mobile and desktop wrapper here be replaced with the visibility component? If it will still require custom styling then LGTM

Reply:
Noted, have made the changes 🙏